Update to PDF.js v3.6.172

mcatanzaro opened the pull request:
#### e456b31b38489ce1b03d87a50a5c7572c8dfa370
<pre>
Update to PDF.js v3.6.172
<a href="https://bugs.webkit.org/show_bug.cgi?id=256469">https://bugs.webkit.org/show_bug.cgi?id=256469</a>

Reviewed by Tim Nguyen.

* Source/ThirdParty/pdfjs/PdfJSFiles.cmake:
* Source/ThirdParty/pdfjs/build/pdf.js:
(return.):
(return):
* Source/ThirdParty/pdfjs/build/pdf.sandbox.js:
* Source/ThirdParty/pdfjs/build/pdf.worker.js:
(return.):
(return):
* Source/ThirdParty/pdfjs/web/debugger.css:
(#PDFBug,):
(#PDFBug table.showText,):
* Source/ThirdParty/pdfjs/web/images/gv-toolbarButton-download.svg: Added.
* Source/ThirdParty/pdfjs/web/images/gv-toolbarButton-openinapp.svg: Added.
* Source/ThirdParty/pdfjs/web/locale/en-US/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/it/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ja/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ko/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/lo/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/nn-NO/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/zh-CN/viewer.properties:
* Source/ThirdParty/pdfjs/web/viewer.css:
(.textLayer :is(span, br)):
(@media screen and (forced-colors: active) .annotationLayer .textWidgetAnnotation :is(input, textarea):required,):
(.annotationLayer :is(.linkAnnotation, .buttonWidgetAnnotation.pushButton) &gt; a):
(.annotationLayer):
(.annotationLayer .textWidgetAnnotation :is(input, textarea),):
(.annotationLayer .textWidgetAnnotation :is(input, textarea):required,):
(.annotationLayer .textWidgetAnnotation :is(input, textarea)[disabled],):
(.annotationLayer .textWidgetAnnotation :is(input, textarea):hover,):
(.annotationLayer .textWidgetAnnotation :is(input, textarea):focus,):
(.annotationLayer .buttonWidgetAnnotation:is(.checkBox, .radioButton) :focus):
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked::before,):
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked::before):
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked::after):
(.annotationLayer .buttonWidgetAnnotation.radioButton input:checked::before):
(.annotationLayer .buttonWidgetAnnotation:is(.checkBox, .radioButton) input):
(.xfaLayer div,):
(:is(.xfaLeft, .xfaRight) &gt; :is(.xfaCaption, .xfaCaptionForCheckButton)):
(:is(.xfaTop, .xfaBottom) &gt; :is(.xfaCaption, .xfaCaptionForCheckButton)):
(:is(.xfaTextfield, .xfaSelect):focus):
(:is(.xfaCheckbox, .xfaRadio):focus):
(:is(.xfaTop, .xfaBottom) &gt; :is(.xfaTextfield, .xfaSelect)):
(:is(.xfaNonInteractive, .xfaDisabled, .xfaReadOnly) :is(input, textarea)):
(.annotationEditorLayer):
(#hiddenCopyElement):
(.pdfViewer:is(.scrollHorizontal, .scrollWrapped),):
(.pdfViewer.removePageBorders,):
(.spread :is(.page, .dummyPage),):
(.spread .page,):
(.pdfViewer.removePageBorders .spread .page,):
(.pdfViewer .page.loadingIcon::after):
(.pdfViewer .page.loading::after):
(.pdfViewer .page:not(.loading)::after):
(:root):
(@media (prefers-color-scheme: dark) :root):
(#outerContainer:is(.sidebarMoving, .sidebarOpen) #sidebarContainer):
(:is(.toolbar, .editorParamsToolbar, .findbar, #sidebarContainer)):
(#loadingBar .progress):
(@keyframes progressIndeterminate):
(#loadingBar.indeterminate .progress .glimmer):
(#outerContainer.sidebarResizing):
(:is(.doorHanger, .doorHangerRight)::after,):
(.doorHanger::after):
(.doorHangerRight::after):
(:is(.doorHanger, .doorHangerRight)::before):
(.doorHanger::before):
(.doorHangerRight::before):
(#findMsg[data-status=&quot;notFound&quot;]):
(:is(#findResultsCount, #findMsg):empty):
(.dialogButton:is(:hover, :focus-visible)):
(.dialogButton:is(:hover, :focus-visible) &gt; span):
(:is(.toolbarButton, .secondaryToolbarButton, .dialogButton)[disabled]):
(.splitToolbarButton &gt; .toolbarButton:is(:hover, :focus-visible),):
(.toolbarButton:is(:hover, :focus-visible)):
(.secondaryToolbarButton:is(:hover, :focus-visible)):
(:is(.toolbarButton, .secondaryToolbarButton).toggled,):
(:is(.toolbarButton, .secondaryToolbarButton).toggled::before):
(:is(.toolbarButton, .secondaryToolbarButton).toggled:hover:active,):
(.dropdownToolbarButton &gt; select:is(:hover, :focus-visible)):
(:is(.toolbarButton, .secondaryToolbarButton, .treeItemToggler)::before,):
(.dropdownToolbarButton:is(:hover, :focus-visible, :active)::after):
(.toolbarButton:is(:hover, :focus-visible)::before,):
(:is(#openFile, #secondaryOpenFile)::before):
(:is(#download, #secondaryDownload)::before):
(a:is(.toolbarButton, .secondaryToolbarButton)[href=&quot;#&quot;]):
(#pageNumber):
(#pageNumber.visiblePageIsLoading):
(#thumbnailView &gt; a:is(:active, :focus)):
(.thumbnail):
(#thumbnailView &gt; a:last-of-type &gt; .thumbnail):
(a:focus &gt; .thumbnail,):
(.thumbnail.selected):
(.thumbnailImage):
(a:focus &gt; .thumbnail &gt; .thumbnailImage,):
(.thumbnail.selected &gt; .thumbnailImage):
(.thumbnail:not([data-loaded]) &gt; .thumbnailImage):
(.treeItemToggler:hover,):
(@media print #printContainer &gt; .printedPage :is(canvas, img)):
(.textLayer span,): Deleted.
(@media screen and (forced-colors: active) .annotationLayer .textWidgetAnnotation input:required,): Deleted.
(.annotationLayer .linkAnnotation &gt; a,): Deleted.
(.annotationLayer .linkAnnotation &gt; a:hover,): Deleted.
(.annotationLayer .textWidgetAnnotation input,): Deleted.
(.annotationLayer .textWidgetAnnotation input:required,): Deleted.
(.annotationLayer .textWidgetAnnotation input[disabled],): Deleted.
(.annotationLayer .textWidgetAnnotation input:hover,): Deleted.
(.annotationLayer .textWidgetAnnotation input:focus,): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox :focus,): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked:before,): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked:before): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked:after): Deleted.
(.annotationLayer .buttonWidgetAnnotation.radioButton input:checked:before): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox input,): Deleted.
(.xfaLayer div): Deleted.
(.xfaLayer svg): Deleted.
(.xfaLayer svg *): Deleted.
(.xfaLeft &gt; .xfaCaption,): Deleted.
(.xfaTop &gt; .xfaCaption,): Deleted.
(.xfaTextfield:focus,): Deleted.
(.xfaCheckbox:focus,): Deleted.
(.xfaTop &gt; .xfaTextfield,): Deleted.
(.xfaNonInteractive input,): Deleted.
(.annotationEditorLayer .freeTextEditor:hover:not(.selectedEditor),): Deleted.
(.pdfViewer .page.loadingIcon:after): Deleted.
(.pdfViewer .page.loading:after): Deleted.
(.pdfViewer .page:not(.loading):after): Deleted.
(.pdfViewer.enablePermissions .textLayer span): Deleted.
(#outerContainer.sidebarMoving #sidebarContainer,): Deleted.
(.toolbar input,): Deleted.
(#outerContainer.sidebarResizing #sidebarContainer,): Deleted.
(.doorHanger:after,): Deleted.
(.doorHanger:after): Deleted.
(.doorHangerRight:after): Deleted.
(.doorHanger:before,): Deleted.
(.doorHanger:before): Deleted.
(.doorHangerRight:before): Deleted.
(#findMsg): Deleted.
(#findResultsCount:empty,): Deleted.
(.dialogButton:hover,): Deleted.
(.dialogButton:hover &gt; span,): Deleted.
(.toolbarButton[disabled],): Deleted.
(.splitToolbarButton &gt; .toolbarButton:hover,): Deleted.
(.toolbarButton:hover,): Deleted.
(.secondaryToolbarButton:hover,): Deleted.
(.toolbarButton.toggled,): Deleted.
(.toolbarButton.toggled::before,): Deleted.
(.toolbarButton.toggled:hover:active,): Deleted.
(.dropdownToolbarButton &gt; select:hover,): Deleted.
(.toolbarButton::before,): Deleted.
(.dropdownToolbarButton:hover::after,): Deleted.
(.toolbarButton:hover::before,): Deleted.
(#openFile::before,): Deleted.
(#download::before,): Deleted.
(a.toolbarButton[href=&quot;#&quot;],): Deleted.
(#thumbnailView &gt; a:active,): Deleted.
(#thumbnailView &gt; a:last-of-type &gt; .thumbnail:not([data-loaded])): Deleted.
(.thumbnail:not([data-loaded])): Deleted.
(.thumbnailSelectionRing): Deleted.
(a:focus &gt; .thumbnail &gt; .thumbnailSelectionRing &gt; .thumbnailImage,): Deleted.
(a:focus &gt; .thumbnail &gt; .thumbnailSelectionRing,): Deleted.
(.thumbnail.selected &gt; .thumbnailSelectionRing &gt; .thumbnailImage): Deleted.
(.thumbnail.selected &gt; .thumbnailSelectionRing): Deleted.
(@media print #printContainer &gt; .printedPage canvas,): Deleted.
* Source/ThirdParty/pdfjs/web/viewer.js:
(__webpack_require__.const.PDFViewerApplication.async _initializeViewerComponents):
(__webpack_require__.const.PDFViewerApplication.async download):
(__webpack_require__):
(__webpack_require__.const.PDFViewerApplication.async save): Deleted.
(__webpack_require__.const.PDFViewerApplication.downloadOrSave): Deleted.
(__webpack_require__.const.PDFViewerApplication._documentError): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError): Deleted.
(__webpack_require__.const.PDFViewerApplication.progress): Deleted.
(__webpack_require__.const.PDFViewerApplication.load.firstPagePromise.then): Deleted.
(__webpack_require__.const.PDFViewerApplication.load.async pagesPromise): Deleted.
(__webpack_require__.const.PDFViewerApplication.load): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _scriptingDocProperties): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeAutoPrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeMetadata): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializePageLabels): Deleted.
(__webpack_require__.const.PDFViewerApplication._initializePdfHistory): Deleted.
(__webpack_require__.const.PDFViewerApplication._initializeAnnotationStorageCallbacks): Deleted.
(__webpack_require__.const.PDFViewerApplication.setInitialView): Deleted.
(__webpack_require__.const.PDFViewerApplication._cleanup): Deleted.
(__webpack_require__.const.PDFViewerApplication.forceRendering): Deleted.
(__webpack_require__.const.PDFViewerApplication.beforePrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.afterPrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.rotatePages): Deleted.
(__webpack_require__.const.PDFViewerApplication.requestPresentationMode): Deleted.
(__webpack_require__.const.PDFViewerApplication.triggerPrinting): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindWindowEvents.addWindowResolutionChange._boundEvents.removeWindowResolutionChange): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindWindowEvents.addWindowResolutionChange): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindWindowEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.unbindEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.unbindWindowEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication._accumulateTicks): Deleted.
(__webpack_require__.const.PDFViewerApplication._accumulateFactor): Deleted.
(__webpack_require__.const.PDFViewerApplication._centerAtPos): Deleted.
(__webpack_require__.const.PDFViewerApplication._unblockDocumentLoadEvent): Deleted.
(__webpack_require__.const.PDFViewerApplication.get scriptingReady): Deleted.
(__webpack_require__.validateFileURL): Deleted.
(__webpack_require__.async loadFakeWorker): Deleted.
(__webpack_require__.async loadPDFBug): Deleted.
(__webpack_require__.reportPageStatsPDFBug): Deleted.

Canonical link: <a href="https://commits.webkit.org/263819@main">https://commits.webkit.org/263819@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a201401df039604204eca055a38b167643012f53

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/5797 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/5963 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/6151 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/7357 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/6190 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/6185 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/5930 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/7974 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/5903 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/5951 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/5247 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/7412 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/3428 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/5226 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/13190 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/5296 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/5304 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/7505 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/5749 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/4721 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/5192 "Built successfully") | | | 
| [❌ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/1376 "Failed to push commit to Webkit repository") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/9311 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/5553 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/e77a99ef82232e0fa9201c59cd0bc30d9be1f5f6. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/13581

mcatanzaro committed:
Update to PDF.js v3.6.172
https://bugs.webkit.org/show_bug.cgi?id=256469

Reviewed by Tim Nguyen.

* Source/ThirdParty/pdfjs/PdfJSFiles.cmake:
* Source/ThirdParty/pdfjs/build/pdf.js:
(return.):
(return):
* Source/ThirdParty/pdfjs/build/pdf.sandbox.js:
* Source/ThirdParty/pdfjs/build/pdf.worker.js:
(return.):
(return):
* Source/ThirdParty/pdfjs/web/debugger.css:
(#PDFBug,):
(#PDFBug table.showText,):
* Source/ThirdParty/pdfjs/web/images/gv-toolbarButton-download.svg: Added.
* Source/ThirdParty/pdfjs/web/images/gv-toolbarButton-openinapp.svg: Added.
* Source/ThirdParty/pdfjs/web/locale/en-US/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/it/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ja/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ko/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/lo/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/nn-NO/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/zh-CN/viewer.properties:
* Source/ThirdParty/pdfjs/web/viewer.css:
(.textLayer :is(span, br)):
(@media screen and (forced-colors: active) .annotationLayer .textWidgetAnnotation :is(input, textarea):required,):
(.annotationLayer :is(.linkAnnotation, .buttonWidgetAnnotation.pushButton) > a):
(.annotationLayer):
(.annotationLayer .textWidgetAnnotation :is(input, textarea),):
(.annotationLayer .textWidgetAnnotation :is(input, textarea):required,):
(.annotationLayer .textWidgetAnnotation :is(input, textarea)[disabled],):
(.annotationLayer .textWidgetAnnotation :is(input, textarea):hover,):
(.annotationLayer .textWidgetAnnotation :is(input, textarea):focus,):
(.annotationLayer .buttonWidgetAnnotation:is(.checkBox, .radioButton) :focus):
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked::before,):
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked::before):
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked::after):
(.annotationLayer .buttonWidgetAnnotation.radioButton input:checked::before):
(.annotationLayer .buttonWidgetAnnotation:is(.checkBox, .radioButton) input):
(.xfaLayer div,):
(:is(.xfaLeft, .xfaRight) > :is(.xfaCaption, .xfaCaptionForCheckButton)):
(:is(.xfaTop, .xfaBottom) > :is(.xfaCaption, .xfaCaptionForCheckButton)):
(:is(.xfaTextfield, .xfaSelect):focus):
(:is(.xfaCheckbox, .xfaRadio):focus):
(:is(.xfaTop, .xfaBottom) > :is(.xfaTextfield, .xfaSelect)):
(:is(.xfaNonInteractive, .xfaDisabled, .xfaReadOnly) :is(input, textarea)):
(.annotationEditorLayer):
(#hiddenCopyElement):
(.pdfViewer:is(.scrollHorizontal, .scrollWrapped),):
(.pdfViewer.removePageBorders,):
(.spread :is(.page, .dummyPage),):
(.spread .page,):
(.pdfViewer.removePageBorders .spread .page,):
(.pdfViewer .page.loadingIcon::after):
(.pdfViewer .page.loading::after):
(.pdfViewer .page:not(.loading)::after):
(:root):
(@media (prefers-color-scheme: dark) :root):
(#outerContainer:is(.sidebarMoving, .sidebarOpen) #sidebarContainer):
(:is(.toolbar, .editorParamsToolbar, .findbar, #sidebarContainer)):
(#loadingBar .progress):
(@keyframes progressIndeterminate):
(#loadingBar.indeterminate .progress .glimmer):
(#outerContainer.sidebarResizing):
(:is(.doorHanger, .doorHangerRight)::after,):
(.doorHanger::after):
(.doorHangerRight::after):
(:is(.doorHanger, .doorHangerRight)::before):
(.doorHanger::before):
(.doorHangerRight::before):
(#findMsg[data-status="notFound"]):
(:is(#findResultsCount, #findMsg):empty):
(.dialogButton:is(:hover, :focus-visible)):
(.dialogButton:is(:hover, :focus-visible) > span):
(:is(.toolbarButton, .secondaryToolbarButton, .dialogButton)[disabled]):
(.splitToolbarButton > .toolbarButton:is(:hover, :focus-visible),):
(.toolbarButton:is(:hover, :focus-visible)):
(.secondaryToolbarButton:is(:hover, :focus-visible)):
(:is(.toolbarButton, .secondaryToolbarButton).toggled,):
(:is(.toolbarButton, .secondaryToolbarButton).toggled::before):
(:is(.toolbarButton, .secondaryToolbarButton).toggled:hover:active,):
(.dropdownToolbarButton > select:is(:hover, :focus-visible)):
(:is(.toolbarButton, .secondaryToolbarButton, .treeItemToggler)::before,):
(.dropdownToolbarButton:is(:hover, :focus-visible, :active)::after):
(.toolbarButton:is(:hover, :focus-visible)::before,):
(:is(#openFile, #secondaryOpenFile)::before):
(:is(#download, #secondaryDownload)::before):
(a:is(.toolbarButton, .secondaryToolbarButton)[href="#"]):
(#pageNumber):
(#pageNumber.visiblePageIsLoading):
(#thumbnailView > a:is(:active, :focus)):
(.thumbnail):
(#thumbnailView > a:last-of-type > .thumbnail):
(a:focus > .thumbnail,):
(.thumbnail.selected):
(.thumbnailImage):
(a:focus > .thumbnail > .thumbnailImage,):
(.thumbnail.selected > .thumbnailImage):
(.thumbnail:not([data-loaded]) > .thumbnailImage):
(.treeItemToggler:hover,):
(@media print #printContainer > .printedPage :is(canvas, img)):
(.textLayer span,): Deleted.
(@media screen and (forced-colors: active) .annotationLayer .textWidgetAnnotation input:required,): Deleted.
(.annotationLayer .linkAnnotation > a,): Deleted.
(.annotationLayer .linkAnnotation > a:hover,): Deleted.
(.annotationLayer .textWidgetAnnotation input,): Deleted.
(.annotationLayer .textWidgetAnnotation input:required,): Deleted.
(.annotationLayer .textWidgetAnnotation input[disabled],): Deleted.
(.annotationLayer .textWidgetAnnotation input:hover,): Deleted.
(.annotationLayer .textWidgetAnnotation input:focus,): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox :focus,): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked:before,): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked:before): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox input:checked:after): Deleted.
(.annotationLayer .buttonWidgetAnnotation.radioButton input:checked:before): Deleted.
(.annotationLayer .buttonWidgetAnnotation.checkBox input,): Deleted.
(.xfaLayer div): Deleted.
(.xfaLayer svg): Deleted.
(.xfaLayer svg *): Deleted.
(.xfaLeft > .xfaCaption,): Deleted.
(.xfaTop > .xfaCaption,): Deleted.
(.xfaTextfield:focus,): Deleted.
(.xfaCheckbox:focus,): Deleted.
(.xfaTop > .xfaTextfield,): Deleted.
(.xfaNonInteractive input,): Deleted.
(.annotationEditorLayer .freeTextEditor:hover:not(.selectedEditor),): Deleted.
(.pdfViewer .page.loadingIcon:after): Deleted.
(.pdfViewer .page.loading:after): Deleted.
(.pdfViewer .page:not(.loading):after): Deleted.
(.pdfViewer.enablePermissions .textLayer span): Deleted.
(#outerContainer.sidebarMoving #sidebarContainer,): Deleted.
(.toolbar input,): Deleted.
(#outerContainer.sidebarResizing #sidebarContainer,): Deleted.
(.doorHanger:after,): Deleted.
(.doorHanger:after): Deleted.
(.doorHangerRight:after): Deleted.
(.doorHanger:before,): Deleted.
(.doorHanger:before): Deleted.
(.doorHangerRight:before): Deleted.
(#findMsg): Deleted.
(#findResultsCount:empty,): Deleted.
(.dialogButton:hover,): Deleted.
(.dialogButton:hover > span,): Deleted.
(.toolbarButton[disabled],): Deleted.
(.splitToolbarButton > .toolbarButton:hover,): Deleted.
(.toolbarButton:hover,): Deleted.
(.secondaryToolbarButton:hover,): Deleted.
(.toolbarButton.toggled,): Deleted.
(.toolbarButton.toggled::before,): Deleted.
(.toolbarButton.toggled:hover:active,): Deleted.
(.dropdownToolbarButton > select:hover,): Deleted.
(.toolbarButton::before,): Deleted.
(.dropdownToolbarButton:hover::after,): Deleted.
(.toolbarButton:hover::before,): Deleted.
(#openFile::before,): Deleted.
(#download::before,): Deleted.
(a.toolbarButton[href="#"],): Deleted.
(#thumbnailView > a:active,): Deleted.
(#thumbnailView > a:last-of-type > .thumbnail:not([data-loaded])): Deleted.
(.thumbnail:not([data-loaded])): Deleted.
(.thumbnailSelectionRing): Deleted.
(a:focus > .thumbnail > .thumbnailSelectionRing > .thumbnailImage,): Deleted.
(a:focus > .thumbnail > .thumbnailSelectionRing,): Deleted.
(.thumbnail.selected > .thumbnailSelectionRing > .thumbnailImage): Deleted.
(.thumbnail.selected > .thumbnailSelectionRing): Deleted.
(@media print #printContainer > .printedPage canvas,): Deleted.
* Source/ThirdParty/pdfjs/web/viewer.js:
(__webpack_require__.const.PDFViewerApplication.async _initializeViewerComponents):
(__webpack_require__.const.PDFViewerApplication.async download):
(__webpack_require__):
(__webpack_require__.const.PDFViewerApplication.async save): Deleted.
(__webpack_require__.const.PDFViewerApplication.downloadOrSave): Deleted.
(__webpack_require__.const.PDFViewerApplication._documentError): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError): Deleted.
(__webpack_require__.const.PDFViewerApplication.progress): Deleted.
(__webpack_require__.const.PDFViewerApplication.load.firstPagePromise.then): Deleted.
(__webpack_require__.const.PDFViewerApplication.load.async pagesPromise): Deleted.
(__webpack_require__.const.PDFViewerApplication.load): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _scriptingDocProperties): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeAutoPrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeMetadata): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializePageLabels): Deleted.
(__webpack_require__.const.PDFViewerApplication._initializePdfHistory): Deleted.
(__webpack_require__.const.PDFViewerApplication._initializeAnnotationStorageCallbacks): Deleted.
(__webpack_require__.const.PDFViewerApplication.setInitialView): Deleted.
(__webpack_require__.const.PDFViewerApplication._cleanup): Deleted.
(__webpack_require__.const.PDFViewerApplication.forceRendering): Deleted.
(__webpack_require__.const.PDFViewerApplication.beforePrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.afterPrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.rotatePages): Deleted.
(__webpack_require__.const.PDFViewerApplication.requestPresentationMode): Deleted.
(__webpack_require__.const.PDFViewerApplication.triggerPrinting): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindWindowEvents.addWindowResolutionChange._boundEvents.removeWindowResolutionChange): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindWindowEvents.addWindowResolutionChange): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindWindowEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.unbindEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.unbindWindowEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication._accumulateTicks): Deleted.
(__webpack_require__.const.PDFViewerApplication._accumulateFactor): Deleted.
(__webpack_require__.const.PDFViewerApplication._centerAtPos): Deleted.
(__webpack_require__.const.PDFViewerApplication._unblockDocumentLoadEvent): Deleted.
(__webpack_require__.const.PDFViewerApplication.get scriptingReady): Deleted.
(__webpack_require__.validateFileURL): Deleted.
(__webpack_require__.async loadFakeWorker): Deleted.
(__webpack_require__.async loadPDFBug): Deleted.
(__webpack_require__.reportPageStatsPDFBug): Deleted.

Canonical link: https://commits.webkit.org/263819@main
